practice-two/src/style/sections/testinomials.css

```diff
@@ -0,0 +1,175 @@
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.testimonials__wrapper {
+  padding-bottom: 30px;
+}
+
+.testimonials__header-text {
+  text-align: center;
+  padding-top: 28px;
+}
+
+.testimonials__description {
+  position: relative;
+  padding-top: 56px;
+  width: 240px;
+}
+
+.testimonials__description-text {
+  font-size: var(--font-size-xs);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+  line-height: var(--line-height-base);
+}
+
+.testimonials__description::before {
+  content: '"';
+  font-size: var(--font-size-3xlg);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+  line-height: var(--line-height-normal);
+  color: var(--color-primary);
+  position: absolute;
+  left: 50%;
+  top: 10%;
+  transform: translateX(-50%);
+}
+
+.testimonials__item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.testimonials___info-img {
+  border-radius: 50%;
+}
+
+.testimonials___info {
+  display: flex;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.testimonials__contact {
+  display: flex;
+  flex-direction: column;
+  margin-left: 10px;
+  gap: 5px;
+}
+
+.testimonials__link {
+  display: none;
+}
+
+.testimonials__contact-name {
+  font-size: var(--font-size-xs);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+}
+
+.testimonials__contact-mail {
+  font-size: var(--font-size-2xs);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+}
+
+@media (min-width: 768px) {
+  .testimonials__wrapper {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .testimonials__link {
+    display: block;
+
+    &:hover {
+      transform: scale(1.1);
+    }
+  }
+
+  .testimonials__header {
+    margin-top: 40px;
+  }
+
+  .testimonials__content {
+    display: flex;
+    align-items: center;
+    gap: 40px;
+    margin-top: 24px;
+  }
+
+  .testimonials__description {
+    width: 470px;
+  }
+
+  .testimonials__description-text {
+    font-size: var(--font-size-xs);
+    line-height: var(--line-height-base);
+  }
+
+  .testimonials__icon {
+    width: 40px;
+    height: 40px;
+    fill: none;
+  }
+}
+
+@media (min-width: 1440px) {
+  .testimonials {
+    padding-top: 90px;
+    padding-bottom: 70px;
+    gap: 40px;
+  }
+
+  .testimonials__description::before {
+    --font-size: 144px;
+    font-size: var(--font-size);
+    top: -10%;
+  }
+
+  .testimonials__content {
+    margin-top: 50px;
+    gap: 130px;
+  }
+
+  .testimonials__description {
+    width: 750px;
+  }
+
+  .testimonials__description-text {
+    font-size: var(--font-size-lg);
+    line-height: var(--font-size-2xlg);
+  }
+
+  .testimonials__icon {
+    width: 64px;
+    height: 64px;
+  }
+
+  .testimonials___info {
+    margin-top: 46px;
+  }
+
+  .testimonials___info-img {
+    width: 72px;
+    height: 72px;
+  }
+
+  .person-img {
+    width: 100%;
```

Comment:
You should create a common class for image, you apply this style for all image in your app